open-api-spec.yaml

```diff
@@ -810,32 +840,43 @@ components:
           type: string
           description: A reference that can be used by external systems to reconcile this payment with their systems. E.g. An invoice number. (Optional)
           example: 'customer: 847458475'
+        createdAt:
+          type: string
+          format: date-time
+          pattern: '2022-03-12T23:20:50.52Z'
+          description: The date and time when the outgoing payment was created.
+        updatedAt:
+          type: string
+          format: date-time
+          pattern: '2022-04-01T10:24:36.11Z'
+          description: The date and time when the outgoing payment was updated.
     amount:
       title: Amount
       type: object
       description: |-
-        All amounts in open payments are represented as an unsigned 64-bit integer amount and an asset code and scale.
+        All amounts in open payments are represented as a value and an asset code and scale.
+
+        The `value` is a positive monetary value, represented as a string. The following regular expression is an implementation of this definition: `^[1-9]\d*$`.
 
         The `assetCode` is a code that indicates the underlying asset. In most cases this SHOULD be a 3-character ISO 4217 currency code.
 
-        The `assetScale` indicates how the value of `amount` has been scaled relative to the natural scale of the asset. For example, an `amount` of `1234` with an `assetScale` of `2` represents an amount of `12.34`.
+        The `assetScale` indicates how the `value` has been scaled relative to the natural scale of the asset. For example, an `value` of `"1234"` with an `assetScale` of `2` represents an amount of 12.34.
       examples:
-        - amount: 2500
+        - value: '2500'
           assetCode: USD
           assetScale: 2
-        - amount: 500
+        - value: '500'
           assetCode: EUR
           assetScale: 2
-        - amount: 123456789
+        - value: '123456789'
           assetCode: BTC
           assetScale: 8
       properties:
-        amount:
-          type: integer
+        value:
+          type: string
```

Comment:
Could we add a custom(?) `uint64` format like here:
https://developers.google.com/discovery/v1/type-format

Reply:
Actually, we can.